collector/kernel/bpf_src/render_bpf.c

```diff
@@ -1738,93 +1672,6 @@ int on_tcp_reset(struct pt_regs *ctx, struct sock *sk)
   return 0;
 }
 
-int on_consume_skb(struct pt_regs *ctx, struct sk_buff *skb)
```

Comment:
Interesting.  We always return a '0' on error or not.  Is this a common pattern in the collector?

Reply:
Not in the collector as a whole, just in the ebpf code for probe functions.